timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosRemoteClients.java

```diff
@@ -83,4 +101,16 @@
                         _unused -> ImmutableMap.of()))
                 .collect(Collectors.toList());
     }
+
+    @Path("/" + PaxosTimeLockConstants.INTERNAL_NAMESPACE
```

Comment:
nit: extract constant?

Reply:
meh, don't think it's worth it, I find it easier to parse the path here in terms of how things are grouped together

i.e. is it clientPaxos vs leaderPaxos, given the use case is it batch or just normal etc.